_aggregations/metric/scripted-metric.md

```diff
@@ -9,44 +9,220 @@ redirect_from:
 
 # Scripted metric aggregations
 
-The `scripted_metric` metric is a multi-value metric aggregation that returns metrics calculated from a specified script.
+The `scripted_metric` aggregation is a multi-value metric aggregation that returns metrics calculated from a specified script. The aggregation goes through up to four script stages during execution. These stages run in order and allow you to accumulate and combine results from your documents.
```

Comment:
`The aggregation goes through up to four script stages during execution. These stages run in order and allow you to accumulate and combine results from your documents.`

I find this wording confusing - this wording seems to imply that upto 4 independent scripts can run at max. 

Something like this might be clear: `A script has four stages: <name the stages>, which are run in order by each aggregation, and allow you to combine results from your documents.`